en/docs/developer-portal/test-api.md

```diff
@@ -1,30 +1,31 @@
 # Test an API
  
-Before using your API in production, you can test it using the integrated OpenAPI console in Choreo. Once the API is deployed in the required environment, an API consumer can use the application credentials, generate an OAuth 2.0 token, and invoke the API to test its functionality by providing values for the required parameters. 
+Before using your API in production, you can test it using the integrated console in Choreo. Once you deploy your API in the required environment, an API consumer can use the application credentials, generate an OAuth 2.0 token, and invoke the API to test its functionality by providing values for the required parameters. 
 
 This guide will take you through the steps to test an API in Choreo. 
 
-1. Sign in to Choreo Developer Portal at [https://devportal.choreo.dev](https://devportal.choreo.dev). Alternatively, click  Developer Portal link on the Choreo console header. 
+1. Sign in to Choreo Developer Portal at [https://devportal.choreo.dev](https://devportal.choreo.dev). Alternatively, click the Developer Portal link on the Choreo console header. 
```

Comment:
```suggestion
```suggestion
1. Sign in to Choreo Developer Portal at [https://devportal.choreo.dev](https://devportal.choreo.dev). 

    Alternatively, click **Developer Portal** on the Choreo Console header.
```